### PR TITLE
[IOCOM-1798] Scale up app-messages (redis, function app) and remote-content configurations container

### PR DESCRIPTION
**Pull request by lucacavallaro**
#### NEW io-p-weu-com-rg-01 (Resource Group) 🟢
- location: `westeurope`

#### NEW io-p-weu-com-citizen-func-01|02 (Azure Function App) 🟢
- sku: `P2mv3`
- minimum instances: `20`
- resource group: `io-p-weu-com-rg-01`

#### io-p-redis-app-messages-std-v6 (Azure Cache for Redis) 🟡
- family from `C0` to `C2`
 
#### io-p-messages-remote-content (Azure CosmosDB) 🟡
- `message-configuration` RU from `2_000` to `10_000`

### Type of changes

- [x] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Env to apply

- [ ] DEV
- [ ] UAT
- [x] PROD

### Does this introduce a change to production resources with possible user impact?

- [x] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### How to apply

After PR is approved
1. run deploy pipeline from Azure DevOps [io-platform-iac-projects](https://dev.azure.com/pagopaspa/io-platform-iac-projects/_build?view=folders)
2. select PR branch
3. wait for approval

### CODE REVIEW

**Comment by lucacavallaro** [comment]
@Krusty93 @christian-calabrese Static Analysis fails with an error on `core` module ☹️

**Review by christian-calabrese — APPROVED** [review]
LGTM

File: src/domains/messages-app/10_function_messages_xl.tf [hunk 1]
```diff
@@ -0,0 +1,598 @@
...
+
+  allowed_ips = concat(
+    [],
+    local.app_insights_ips_west_europe,
+  )
+
+  # Action groups for alerts
+  action = [
+    {
+      action_group_id    = data.azurerm_monitor_action_group.error_action_group.id
```
Comment (Krusty93) [inline src/domains/messages-app/10_function_messages_xl.tf:86]: No domain specific action group here?

File: src/domains/messages-app/10_function_messages_xl.tf [hunk 2]
```diff
@@ -0,0 +1,598 @@
...
+      webhook_properties = {}
+    }
+  ]
+
+  client_certificate_mode = "Required"
```
Comment (Krusty93) [inline src/domains/messages-app/10_function_messages_xl.tf:91]: Sure? I'd remove this one

Comment (lucacavallaro) [reply src/domains/messages-app/10_function_messages_xl.tf:98]: I just copied the old file 😢 are you sure that we can safely remove this property?

Comment (Krusty93) [reply src/domains/messages-app/10_function_messages_xl.tf:102]: Yep

**Review by Krusty93 — APPROVED** [review]

**Pull request merged**